### DISCUSSION TITLE
Stone Walls

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2046] [Ideas]
Again these are rendered on the Organic Maps but are so faint it is also impossible to see them. In NW England these walls are very sturdy and impossible to climb over and therefore it would be great if the thin black line could be made more obvious and prominent. This would be very beneficial for Walkers and Hikers.

Best wishes

**Comment by biodranik** [comment]
Can you please share some screenshots in OM and point to some examples in OpenStreetMap.org?

**Comment by pastk** [comment]
Indeed all fences, walls, etc. are displayed by a very thin dotted line which is hard to see
![Screenshot_1644352467](https://user-images.githubusercontent.com/18434508/153071210-0e4f163a-339a-4135-ba0a-881d2820b2e8.png)

**Reply by pwb999** [reply]
Yes. The walls are shown but only faintly visible.
I live in Cumbria England with miles of stone walls. You cannot cross these walls easily and therefore you need to know where they are before you decide to go on a hike. I have therefore extracted barrier=wall from OSM and overlaid this version in BLACK Width 2 so I can see them. However I would like to change the visibility of this bookmark file to say 10 but if i change the Visibility Tag nothing happens. ie they are all visible whatever the Zoom level?

**Reply by biodranik** [reply]
visibility tag in bookmarks is either 1 (show the bookmark) or 0 (hide it).

**Comment by pastk** [comment]
I've created an issue to improve the rendering https://github.com/organicmaps/organicmaps/issues/2063

**Comment by pwb999** [comment]
This is an image of one of many. Very old and preserved so you can only cross where there is a STILE or GATE. Also both of these OSM Tags are in OM but very very tiny to see.
![cumbrian-stone-wall-autumn-lake-district-manmade-top-mountain-national-park-united-kingdom-164114026](https://user-images.githubusercontent.com/58628197/154257565-c9bc1089-4c0c-4c8c-a786-14935bb93b27.jpeg)
